pkg/config/setup/system_probe.go

```diff
@@ -194,6 +194,7 @@ func InitSystemProbeConfig(cfg pkgconfigmodel.Config) {
 	cfg.BindEnv(join(netNS, "max_failed_connections_buffered"))
 	cfg.BindEnvAndSetDefault(join(spNS, "closed_connection_flush_threshold"), 0)
 	cfg.BindEnvAndSetDefault(join(spNS, "closed_channel_size"), 500)
+	cfg.BindEnvAndSetDefault(join(netNS, "closed_buffer_wakeup_count"), 5)
```

Comment:
What is the plan to migrate other perf-buffers to this technique?
Do we plan to create a different configuration per perf-buffer?
Maybe we should have a single configuration for all perf-buffers, and allow different teams to create a dedicate configuration to override it
```go
cfg.BindEnvAndSetDefault(join(spNS, "common_wakeup_count"), 5)
cfg.BindEnv(join(netNS, "closed_buffer_wakeup_count"))
```
in `adjust_npm.go`
```go
	applyDefault(cfg, netNS("closed_buffer_wakeup_count"), cfg.GetInt(spNS("common_wakeup_count")))
```

Reply:
> What is the plan to migrate other perf-buffers to this technique?

I was keeping each team to a separate PR. I wanted to consult first to ensure it was actually a feature they wanted.

> Do we plan to create a different configuration per perf-buffer?

Yes, because how much you want to keep in the buffer before wakeup is a usecase-specific value.

Reply:
That sounds like something that could be addressed in future PR by NPM folks. That is additional complexity that I don't think is necessary for this PR, which is trying to closely match the behavior from the custom batching.